src/System.Threading/tests/SemaphoreSlimTests.cs

```diff
@@ -160,27 +160,6 @@ public static void RunSemaphoreSlimTest7_AvailableWaitHandle()
             RunSemaphoreSlimTest7_AvailableWaitHandle(0, 10, SemaphoreSlimActions.Release, true);
         }
 
-        [Fact]
-        [OuterLoop]
```

Comment:
These all used to be outer loop tests and are now inner loop. Just to clarify, these are fast enough to be inner-loop and were probably just outer-loop because it was really 10 tests packed into 1?